config/popular-email-domains.js

```diff
@@ -2,51 +2,28 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-// This is a list of popular email domains per
-// https://github.com/mailcheck/mailcheck/wiki/List-of-Popular-Domains
+// This is a list of popular email domains based on FxA users
```

Comment:
nit: could you please include a link to the bug in this comment, in case we need to refresh the list in future?